# Hypothesis Testing Q4

In [1]:
import pandas as pd
import numpy as np
from scipy import stats


# Data description


α == 0.05 (95% Confidence)

Y == Discrete
X == Discrete

Since there are more than 2 variable we will perform Chi-Square test

# Chi-Square Test

In [2]:
df = pd.read_csv(r"C:\Users\Shivam\Downloads\Costomer+OrderForm.csv")

In [3]:
stats.chi2_contingency([df['Phillippines'].value_counts(), df['Indonesia'].value_counts(), df['Malta'].value_counts(), df['India'].value_counts()])

(3.8589606858203545,
 0.2771020991233144,
 3,
 array([[271.75,  28.25],
        [271.75,  28.25],
        [271.75,  28.25],
        [271.75,  28.25]]))

P value of Chi-Square test == 0.277 > α

Thus HO is accepted.
The defective % does not varies significantly by centres